### PR TITLE
Contact us send message

### PR DESCRIPTION
**Pull request by MlulekiN**

### CODE REVIEW

File: src/test/java/pages/ContactUsPage.java [hunk 1]
```diff
@@ -4,4 +4,9 @@
 
 public class ContactUsPage {
     public static By ContactUsHeading = By.xpath("//h2[text()='Contact Us']");
+    public static By sendMessageButton = By.xpath("//form/descendant::button[contains(@class,'contact')]");
+    public static By successMessageDisplay = By.xpath("//div[contains(@id,'success')]");
+    public static By sendMessageFormInput(String personalInfo) {
```
Comment (benedictnkosi) [inline src/test/java/pages/ContactUsPage.java]: rename this parameter to placeHolder

File: src/test/java/stepDefinition/ContactUsSteps.java [hunk 2]
```diff
@@ -23,4 +24,23 @@ public void userValidatesThatContactUsPageIsOpen() {
         WebElement contactUsPage = driver.findElement(ContactUsPage.ContactUsHeading);
         Assert.assertTrue(contactUsPage.isDisplayed());
     }
+
+    @When("User clicks on input field {string} and enters {string}")
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/ContactUsSteps.java]: user enters..... no need to mention that there is a click first

File: src/test/java/stepDefinition/ContactUsSteps.java [hunk 3]
```diff
@@ -23,4 +24,23 @@ public void userValidatesThatContactUsPageIsOpen() {
...
+
+    @When("The user clicks on the send message button")
+    public void theUserClicksOnTheSendMessageButton() {
+        SeleniumActions.click(ContactUsPage.sendMessageButton, driver);
+    }
+
+    @Then("User validates message is successfully sent")
+    public void userValidatesMessageIsSuccessfullySent() {
+        SeleniumActions.waitForVisibility(ContactUsPage.successMessageDisplay, driver);
+        WebElement contactUsPage = driver.findElement(ContactUsPage.successMessageDisplay);
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/ContactUsSteps.java]: rename this element, its not a contactUsPage

**Review by benedictnkosi — CHANGES_REQUESTED** [review]

File: src/test/java/stepDefinition/ContactUsSteps.java (hunk 3 above)
Comment (MlulekiN) [reply src/test/java/stepDefinition/ContactUsSteps.java]: Done

File: src/test/java/pages/ContactUsPage.java (hunk 1 above)
Comment (MlulekiN) [reply src/test/java/pages/ContactUsPage.java]: Done

File: src/test/java/stepDefinition/ContactUsSteps.java (hunk 2 above)
Comment (MlulekiN) [reply src/test/java/stepDefinition/ContactUsSteps.java]: Done

**Review by benedictnkosi — APPROVED** [review]

**Pull request merged**